### ISSUE TITLE
Feature Request: Search on both Tweet Location & User Place simultaneously

### ISSUE THREAD

**Issue by Saijin-Naib** [ale-cristofori/geotweet#4]
I'd like to see the ability to search on both Tweet Location & User Place simultaneously. Whether or not the results get put into two different memory layers as they do now is not something I'm terribly fussed over.

Is this possible within the current framework of this plugin?

Thanks!

**Comment by ale-cristofori** [comment]
No, it is not possible. The two searches fire two different stream listeners in the underlying Twitter API. They could be run simultaneously to achieve what you desire but it's actually not implemented in this version of the plug-in.

**Comment by Saijin-Naib** [comment]
Thanks for clarifying. Well, if you ever feel like it is something to implement, I think it'd be a nice UX improvement.
